templates/pages/admin/inventory/agent.html.twig

```diff
@@ -146,5 +148,25 @@
                 })
             ) }}
         {% endfor %}
+
+        <h2 class="header">{{ __('Enabled tasks') }}</h2>
+
+        {% for modulefield, modulelabel in {
+            'use_module_wake_on_lan': __('Wake on LAN'),
+            'use_module_computer_inventory': __('Computer inventory'),
+            'use_module_esx_remote_inventory': __('ESX remote inventory'),
+            'use_module_network_inventory': __('Network inventory (SNMP)'),
+            'use_module_network_discovery': __('Network discovery (SNMP)'),
+            'use_module_package_deployment': __('Package Deployment'),
+            'use_module_collect_data': __('Collect data'),
+            'use_module_remote_inventory': __('Remote inventory'),
+        } %}
+            {{ fields.checkboxField(
```

Comment:
rather a checkbox field which can mislead user (there is maybe a way to enable/disable the module), a `<i class="ti-ti-check">`

Reply:
![image](https://user-images.githubusercontent.com/7335054/198264323-b4ba5a21-20c0-460c-8875-8d63e536b6a9.png)